### DISCUSSION TITLE
ASR : Is BPE ASR model is better in accuracy compared to normal char

### DISCUSSION THREAD

**Discussion by harrypotter90** [NVIDIA-NeMo/NeMo#1703] [Q&A] [answered]
I have been using ASR and there are two variant one is normal and the other is the BPE variant config file

Char: https://github.com/NVIDIA/NeMo/blob/main/examples/asr/experimental/conformer/conformer_char.yaml
BPE: https://github.com/NVIDIA/NeMo/blob/main/examples/asr/experimental/conformer/conformer_bpe.yaml

I know that BPE is for Unicode characters and char is where we can represent transcripts with normal characters.
But will there be more accuracy if I use BPE instead of char for English ASR training?

I have been reading some other articles and I am a bit confused, so the question may be stupid. Please clarify.

**Comment by titu1994** [comment, marked as answer]
Subword automatically will not guarantee you better WER, but it does have a few advantages that help make it possible for models to perform better.

Theres a new tutorial for ASR with subword encoding here 
which you could look into for some info on how subword helps CTC based model training - https://colab.research.google.com/github/NVIDIA/NeMo/blob/main/tutorials/asr/08_ASR_with_Subword_Tokenization.ipynb

**Reply by VahidooX** [reply]
When you use subsampling like the one we have in Conformer, BPE based models may give you better WER.
